stock_location_orderpoint_change_priority/models/stock_location_orderpoint.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from collections import defaultdict
+
+from odoo import models
+from odoo.osv import expression
+
+
+class StockLocationOrderpoint(models.Model):
+
+    _inherit = "stock.location.orderpoint"
+
+    def _find_current_replenishment_moves_for_orderpoint_location(self, products=False):
+        """ """
+        domain = [
+            ("location_dest_id", "child_of", self.location_id.id),
```

Comment:
This should have a `self.ensure_one()` check. Otherwise `self.location_id.id` could fail. 
```suggestion
        self.ensure_one()
        domain = [
            ("location_dest_id", "child_of", self.location_id.id),
```